src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateIntegrationTest.java

```diff
@@ -0,0 +1,165 @@
+package org.broadinstitute.hellbender.tools.walkers.sv;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFHeader;
+import org.apache.commons.lang3.tuple.Pair;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.VariantContextTestUtils;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.*;
+
+
+public class SVAnnotateIntegrationTest extends CommandLineProgramTest {
+    final String inputVCFPath = getToolTestDataDir() + "integration.vcf.gz";
+    final File inputVCF = new File(inputVCFPath);
+    private final String largeFileDirectory = GATKBaseTest.largeFileTestDir + "SVAnnotate/";
+    final File gtf = new File(largeFileDirectory + "MANE.selected.GRCh38.v0.95.select_ensembl_genomic.gtf");
+    final File noncodingElements = new File(largeFileDirectory + "noncoding.selected.hg38.bed.gz");
+
+    final List<String> allAnnotationInfoKeys = Arrays.asList(GATKSVVCFConstants.LOF, GATKSVVCFConstants.PROMOTER,
+            GATKSVVCFConstants.DUP_PARTIAL, GATKSVVCFConstants.NONCODING_BREAKPOINT, GATKSVVCFConstants.NONCODING_SPAN,
+            GATKSVVCFConstants.INV_SPAN, GATKSVVCFConstants.PROMOTER, GATKSVVCFConstants.COPY_GAIN,
+            GATKSVVCFConstants.INTERGENIC, GATKSVVCFConstants.NEAREST_TSS, GATKSVVCFConstants.INT_EXON_DUP,
+            GATKSVVCFConstants.PARTIAL_EXON_DUP, GATKSVVCFConstants.MSV_EXON_OVERLAP, GATKSVVCFConstants.UTR,
+            GATKSVVCFConstants.INTRONIC, GATKSVVCFConstants.TSS_DUP, GATKSVVCFConstants.BREAKEND_EXON);
```

Comment:
It's convention to make hard-coded constants all upper-case with underscores, e.g. `INPUT_VCF_PATH`